.github/runner/set-k8s-secrets.sh

```diff
@@ -2,7 +2,7 @@
 # This script runs in the gh-runner container to set K8s secrets needed by VRO containers.
 
 NAMESPACE=$(cat /run/secrets/kubernetes.io/serviceaccount/namespace)
-: ${TARGET_ENV:=${NAMESPACE#va-abd-rrd-}}
+: "${TARGET_ENV:=${NAMESPACE#va-abd-rrd-}}"
```

Comment:
The quotes provide no added benefit for the [colon operator](https://stackoverflow.com/questions/7444504/explanation-of-colon-operator-in-foo-value). e.g., `: asdf asdf asdf` and `: "asdf asdf asdf"` are both safe commands. The extra quotes just adds noise.